source/process/casestudy.rst

```diff
@@ -14,8 +14,12 @@ The process is approximately as follows:
 1. Customer has completed a deployment of Mattermost and is open to a case study. 
 2. Customer case study contact is introduced to a Mattermost case study coordinator over email and completes a questionnaire (below). 
 3. Mattermost case study coordinator arranges a 30-minute phone or video call with the customer for a discussion around the questionnaire and deployment. 
-4. One to two weeks later a draft case study is sent to the customer for review and approval. 
-5. After customer approval, the case study is published to the Mattermost blog and may be used in other marketing materials to share with other potential customers the benefits of a Mattermost solution.
+4. One to two weeks later a draft case study is sent to the customer for review and approval.
+5. Mattermost case study coordinator politely checks in with the customer every 2 business days for an estimate of the draft case study review time.
+6. The case study coordinator also commissions a banner with a clear heading and a nice background image to be included in the case study blog post.
+7. The case study coordinator creates a draft blog post (unpublished) on the Mattermost staging server. The draft will then be reviewed for formatting, grammar, and so on.
+8. When the customer has completed a review of the draft case study, the draft blog post can be updated to be published. 
```

Comment:
The draft (in a Google doc) sent to the customer is differrent from the blog post draft (in WordPress) - I tried to tweak these steps but the ordering is fine